tests/python/common/test_subgraph.py

```diff
@@ -95,6 +95,49 @@ def test_subgraph():
     assert F.allclose(h, g.ndata["h"])
 
 
+def test_subgraph_relabel_nodes():
```

Comment:
test_subgraph_not_relabel_nodes?

Reply:
Maybe clarify the test_subgraph to test_subgraph_relabel_nodes, and change L62 to g.subgraph(nid, relabel_nodes=True)

Reply:
OK, I have modified these.